pydatalab/pydatalab/routes/v0_1/items.py

```diff
@@ -74,6 +74,45 @@ def dereference_files(file_ids: List[Union[str, ObjectId]]) -> Dict[str, Dict]:
     return results
 
 
+def get_equipment_summary():
+    if not current_user.is_authenticated and not CONFIG.TESTING:
+        return (
+            jsonify(
+                status="error",
+                message="Authorization required to access equipment list.",
+            ),
+            401,
+        )
+
+    _project = {
+        "_id": 0,
+        "item_id": 1,
+        "name": 1,
+        "type": 1,
+        "date": 1,
+        "refcode": 1,
+        "location": 1,
+    }
```

Comment:
I'm wondering if we should move this kind of info to the models themselves, much easier to maintain the list of `summary_fields` as e.g.

```
class Equipment(Item):
    ...
    def _get_summary_fields(self):
        return ("item_id", "name", "type", "date", "refcode", "location")

```
then something like `_project = {f: 1 for f in Equipment._get_summary_fields}`, perhaps always explicitly excluding `_id`.

Should then be straightforward to generate summary routes

Reply:
I like this idea. I was thinking about somehow including in each Field() an attribute like `include_in_summary: Bool`, though I wasn't sure how natural that would work with pydantic/ json schemas

Reply:
I recommend we defer this to a separate PR, since it will involve refactoring across the whole app